packages/protoplugin/src/ecmascript/generated-file.ts

```diff
@@ -283,6 +291,7 @@ type MakeImportStatementFn = (
 function processImports(
   el: El[],
   importerPath: string,
+  rewriteImportPath: RewriteImportPathFn,
```

Comment:
It's a bit inconvenient to carry this function around, but it gives us certainty that we touch all managed import statements.